tutorials/toy_optimizations/igarch.jl

```diff
@@ -0,0 +1,74 @@
+# Poisson autoregression
+cd(@__DIR__)
+using StochasticAD, Distributions
+using Optimisers
+import Random
+Random.seed!(1234)
+PLOT = true
+if PLOT
+    using GLMakie
+end
+
+# Poisson autoregression model accumulating a likelihood for starting λ
+function igarch(a, b, c, n, λ) # todo: this should be a Turing/Tilde model
+    ℓ = logpdf(Exponential(1.0), λ)
+    z = rand(Poisson(λ))
+    ℓ += logpdf(Poisson(λ), z)
+    λ = a + b * z + c * λ
+    for i in 2:n
+        z = rand(Poisson(λ))
+        ℓ += logpdf(Poisson(λ), z)
+        λ = a + b * z + c * λ
```

Comment:
```suggestion
        λ = a + b*z + c*λ
```
JuliaFormatter or the style file is broken, `a + b*z + c*λ` is perfectly fine Julia style and helps to see the precedence of operations